deploy/openshift/quay-app.yaml

```diff
@@ -73,20 +73,49 @@ objects:
 - apiVersion: v1
   kind: Service
   metadata:
-    name: prometheus-proxy-loadbalancer
+    name: quay-load-balancer-proxy-protocol-service
+    annotations:
+      service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: ${AWS_LOAD_BALANCER_CONNECTION_IDLE_TIMEOUT}
+      service.beta.kubernetes.io/aws-load-balancer-proxy-protocol: '*'
   spec:
-    type: LoadBalancer
     ports:
-    - name: prometheus-proxy-loadbalancer
+    - name: http
       protocol: TCP
-      port: 3000
-      targetPort: 3000
+      port: ${{LOADBALANCER_SERVICE_HTTP_PORT}}
+      targetPort: ${{LOADBALANCER_SERVICE_PROXY_TARGET_HTTP_PORT}}
+    - name: https
+      protocol: TCP
+      port: ${{LOADBALANCER_SERVICE_PORT}}
+      targetPort: ${{LOADBALANCER_SERVICE_PROXY_TARGET_PORT}}
+    loadBalancerIP:
+    type: LoadBalancer
     selector:
-      app: prometheus
+      ${{QUAY_APP_COMPONENT_LABEL_KEY}}: ${{QUAY_APP_COMPONENT_LABEL_VALUE}}
+
 - apiVersion: v1
   kind: Service
   metadata:
-    name: quay-load-balancer-proxy-protocol-service
+    name: quay-py3-canary-clusterip-service
```

Comment:
The diff output kinda sucks for this section...

Reply:
Yeah, this section basically adds two services which refer to the py3 canary deployment